templates/circle-form.html

```diff
@@ -29,16 +29,16 @@
     <!-- where the router component attaches -->
     <div id="app" style="display:none;"></div>
 
-    <header class="circle-hero">
-      <div class="circle-hero__inner grid_container--m grid_padded">
-        <h1 class="circle-hero__title grid_separator">Join Our Circle</h1>
-        <p class="circle-hero__intro grid_separator">The Texas Tribune is the only member-supported, digitally focused, nonpartisan news organization that informs and engages with Texans about public policy, politics, government and statewide issues.</p>
-        <p class="circle-hero__cta">Our nonprofit newsroom depends on your support!</p>
-      </div>
-    </header>
+    <!-- hero component -->
+    {% with helperClass="hero--circle print__hide", title="Join Our Circle",
```

Comment:
The BEM structure of `print__hide` has never made sense to me given `hide` isn't actually a sub-element. I'm sure you already noticed that.

Reply:
maybe since we're revisiting naming anyway with the `c-component` namespacing, I'll just change it then. AKA when I'm making a bunch of HTML/CSS commits